doc/source/locale/zh_CN/LC_MESSAGES/development/contributing_codebase.po

```diff
@@ -45,16 +48,21 @@ msgid ""
 "result, that is, we need it to be as *backwards compatible* as possible "
 "to avoid mass breakages."
 msgstr ""
+"此外，重要的是，我们不要对代码进行突然更改"
+"这可能会导致大量用户代码中断"
+"也就是说,我们需要它尽可能向后兼容，以避免大规模中断。"
 
 #: ../../source/development/contributing_codebase.rst:23
 msgid ""
 "Checks (including static type checking) are run by ``pre-commit`` - see "
 ":ref:`here <contributing.pre-commit>` for how to run them."
 msgstr ""
+"检查（包括静态类型检查）由 ``pre-commit`` 运行-在 :ref:`这里 <contributing.pre-commit>` "
+"见如何运行。"
 
 #: ../../source/development/contributing_codebase.rst:30
 msgid "Pre-commit"
-msgstr ""
+msgstr "预提交"
```

Comment:
The term "pre-commit" is a specific Git term that does not require translation. It is recommended to keep it in its original form, as it is widely recognized and understood in the context of Git workflows.